pkg/core/resource_graph.go

```diff
@@ -6,32 +6,32 @@ import (
 
 type (
 	ResourceGraph struct {
-		underlying *graph.Directed[Resource]
+		Underlying *graph.Directed[Resource]
```

Comment:
Did we want to move this back to being hidden?

Reply:
Eventually/soon, but as the code stands now, it wouldn't compile. It's not a hard thing to fix, but if you wanted to pull my changes quickly, this is probably the fastest path. (I have a TODO to roll this back on `templatesCompiler`)